optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/roster/RosterGenerator.java

```diff
@@ -98,19 +99,19 @@ public RosterGenerator(EntityManager entityManager) {
     @PostConstruct
     public void setUpGeneratedData() {
         tenantNameGenerator.predictMaximumSizeAndReset(10);
-        generateRoster(10, 7, false);
-        generateRoster(10, 7 * 4, false);
-        generateRoster(20, 7 * 4, false);
-        generateRoster(40, 7 * 2, false);
-        generateRoster(80, 7 * 4, false);
-        generateRoster(10, 7 * 4, true);
-        generateRoster(20, 7 * 4, true);
-        generateRoster(40, 7 * 2, true);
-        generateRoster(80, 7 * 4, true);
+        generateRoster(10, 7, false, false);
+        generateRoster(10, 7 * 4, false, false);
+        generateRoster(20, 7 * 4, false, true);
+        generateRoster(40, 7 * 2, false, false);
+        generateRoster(80, 7 * 4, false, true);
+        generateRoster(10, 7 * 4, true, false);
+        generateRoster(20, 7 * 4, true, true);
+        generateRoster(40, 7 * 2, true, false);
+        generateRoster(80, 7 * 4, true, true);
     }
 
     @Transactional
-    public Roster generateRoster(int spotListSize, int timeSlotListSize, boolean continuousPlanning) {
+    public Roster generateRoster(int spotListSize, int timeSlotListSize, boolean continuousPlanning, boolean isRotation) {
```

Comment:
I believe all datasets should do rotation:
- Either they do a rotation which defines the shift times and for each shift the rotationEmployee
- Or the do a ration that only defines the shift times, but leaves the rotatinEmployee null.

Reply:
Look like I named the parameter poorly; `isRotation` is basically `assignDefaultEmployee`; if true, all shifts get assigned a rotation employee, otherwise all shifts get a rotation employee of null (in short, it behaves like your second bullet point, all datasets use templates (shifts to repeat each week)).